src/composables/transactionItemList.ts

```diff
@@ -0,0 +1,145 @@
+import { computed } from '@vue/composition-api';
+import {
+  IActiveMultisigTx,
+  IDefaultComposableOptions,
+  IDexContracts,
+  ITokenList,
+  ITransaction,
+  ITx,
+} from '../types';
+import {
+  AETERNITY_CONTRACT_ID,
+  defaultTransactionSortingCallback,
+  getInnerTransaction,
+  isTxDex,
+  pipe,
+  TX_DIRECTION,
+} from '../popup/utils';
+import { usePendingMultisigTransaction } from './pendingMultisigTransaction';
+import { useTransactionAndTokenFilter } from './transactionAndTokenFilter';
+import { i18n } from '../store/plugins/languages';
+
+interface TransactionItemList extends IDefaultComposableOptions {
+  isMultisig?: boolean,
+  token: string,
```

Comment:
I suggest to rename this to `tokenContractId` as the current name suggest this might have something to do with the `IToken` which is totally not true.